meta-android/recipes-core/initrdscripts/initramfs-boot-android_1.0.bb

```diff
@@ -12,6 +12,8 @@ SRC_URI = " \
 PACKAGES = "${PN}"
 FILES_${PN} = "/init /machine.conf /distro.conf"
 
+RDEPENDS_${PN} = "e2fsprogs-e2fsck e2fsprogs-resize2fs"
```

Comment:
@shr-project I mean here, this probably causes that we don't have udevadm anymore